### ISSUE TITLE
Optimize PNG assets

### ISSUE THREAD

**Issue by rfnx** [aramk/crayon-syntax-highlighter#309]
Hello,

PNG assets can be optimized, for example with optipng. I was able to reduce the size by 50% on buttons.png and buttons@2x.png, which are loaded for every visitors, without reducing quality. For some other assets, I was able to divide the size by more than 5 (for example /css/images/thumb_horizontal.png) !

It's very simple with optipng (available on every Linux distribution) : 
`
optipng -o7 -strip all file.png
`
and to easily optimize all png in a directory and subdirectories : 
`
find /path/to/dir -iname '*.png' -exec optipng -backup -o7 -strip all {} \;
`
It is a lot of bytes saved for a very small effort :)

Thanks for this amazing plugin !

**Comment by mohammadg** [comment]
Yeah I also noticed this when optimizing my website using Google Page Speed Insights.

**Comment by rfnx** [comment]
As a side note, I added .woff2 versions of the fonts I use in Crayon, it's lighter than .woff format. But I will maybe open a new issue for this.

**Comment by rfnx** [comment]
Can someone look at this please :) ? It's very easy to do but the new version 2.8.0 still has unoptimised images.
